<ui version="4.0" >
 <class>AdvancedSettingsPage</class>
 <widget class="QWidget" name="AdvancedSettingsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Advanced Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="labelDepth" >
     <property name="text" >
      <string>Calculation Depth</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DoubleSlider" name="sliderDepth" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDetail" >
     <property name="text" >
      <string>Detail Level</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DoubleSlider" name="sliderDetail" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAntiAliasing" >
     <property name="text" >
      <string>Image Anti-Aliasing</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frameAntiAliasing" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>4</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioAANone" >
          <property name="text" >
           <string>None</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAALow" >
          <property name="text" >
           <string>Low</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAAMedium" >
          <property name="text" >
           <string>Medium</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAAHigh" >
          <property name="text" >
           <string>High</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelDefault" >
     <property name="text" >
      <string>Default Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore" >
       <property name="text" >
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStore" >
       <property name="text" >
        <string>Store</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>doubleslider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sliderDepth</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
